src/main/drivers/sound_beeper.c

```diff
@@ -27,40 +27,53 @@
 
 
 #ifdef BEEPER
-
 static IO_t beeperIO = DEFIO_IO(NONE);
 static bool beeperInverted = false;
-
+static uint16_t beeperFrequency = 0;
 #endif
 
+
 void systemBeep(bool onoff)
 {
-#ifndef BEEPER
-    UNUSED(onoff);
+#ifdef BEEPER
+    if(beeperFrequency == 0){
+        IOWrite(beeperIO, beeperInverted ? onoff : !onoff);
+    }else{
```

Comment:
Spacing. Try to use space before / after `{` and `}` if they're on the same line.